### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by monster3d**
- unit test for testing Entities/Audio
- unit test for testing Entities/User

### CODE REVIEW

**Comment by MBoretto** [comment]
Hello, please merge develop in your branch to resolve the conflics

File: tests/unit/Entities/AudioTest.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        $this->record['duration'] = null;
+        new Audio($this->record);
+    }
+
+   /**
+    * Test base stage without duration property
+    *
+    * @expectedException Longman\TelegramBot\Exception\TelegramException
+    */
+    public function testBaseStageWithoutFileId()
```
Comment (noplanman) [inline tests/unit/Entities/AudioTest.php]: Same as `testBaseStageWithoutDuration()`, needs to be fixed.
Also docblock needs updating.

**Comment by noplanman** [comment]
Thanks for the PR @monster3d 

After you fix the duplicate test, I'm ok to merge this 👍

File: tests/unit/Entities/AudioTest.php (hunk 1 above)
Comment (monster3d) [inline tests/unit/Entities/AudioTest.php]: Thank you.

**Pull request merged**